src/ucp/rma/flush.c

```diff
@@ -225,18 +227,16 @@ ucs_status_t ucp_ep_flush_progress_pending(uct_pending_req_t *self)
                                           req, 0, &req->send.flush.prog_id);
     }
 
-    if ((status == UCS_OK) || (status == UCS_INPROGRESS)) {
+    if (status == UCS_ERR_NO_RESOURCE) {
+        return UCS_ERR_NO_RESOURCE;
+    } else if (!UCS_STATUS_IS_ERR(status)) {
         /* flushed callback might release the request */
         if (!completed) {
             req->send.lane = UCP_NULL_LANE;
         }
-        return UCS_OK;
-    } else if (status == UCS_ERR_NO_RESOURCE) {
-        return UCS_ERR_NO_RESOURCE;
-    } else {
-        ucp_ep_flush_error(req, status);
-        return UCS_OK;
     }
+
```

Comment:
removed ucp_ep_flush_error?

Reply:
yes, it was moved above, it has to be done prior `ucp_ep_flush_progress` and checking completion of the request